facefusion/core.py

```diff
@@ -328,6 +329,9 @@ def process_step(job_id : str, step_index : int, step_args : Args) -> bool:
 def conditional_process() -> ErrorCode:
 	start_time = time()
 
+	if not state_manager.get_item('workflow'):
```

Comment:
it should never be empty, but use 'auto' instead and make it the default

Reply:
ok